res/values/themes.xml

```diff
@@ -137,4 +137,10 @@
     <item name="emoji_button_bg">#ffffff</item>
     <item name="emoji_color">#000000</item>
   </style>
+  
+<style name="Darkv2" parent="@style/BaseTheme"><item name="android:isLightTheme">false</item><item name="colorKeyboard">#ff121212</item><item name="colorKey">#ff434343</item><item name="colorKeyActivated">#ff1a237e</item><item name="colorLabel">#ffffffff</item><item name="colorLabelActivated">#ff000000</item><item name="colorLabelLocked">#ffffffff</item><item name="secondaryLightOffset">-0.2</item><item name="colorSubLabel">#ffaaaaaa</item><item name="keyBorderWidth">0.0dip</item><item name="keyBorderWidthActivated">0.0dip</item><item name="keyBorderColorBottom">#ff121212</item><item name="emoji_button_bg">?colorKeyActivated</item><item name="emoji_color">#ffffffff</item></style>
+<style name="Lightv2" parent="@style/BaseTheme"><item name="android:isLightTheme">true</item><item name="colorKeyboard">#fff2f5ff</item><item name="colorKey">#ffffffff</item><item name="colorKeyActivated">#ffc2e7ff</item><item name="colorLabel">#ff222222</item><item name="colorLabelActivated">#ff000000</item><item name="colorLabelLocked">#ffffffff</item><item name="secondaryLightOffset">0.0</item><item name="colorSubLabel">#ff333333</item><item name="keyBorderWidth">0.0dip</item><item name="keyBorderWidthActivated">0.0dip</item><item name="keyBorderColorLeft">#ffcccccc</item><item name="keyBorderColorTop">#ffeeeeee</item><item name="keyBorderColorRight">#ffcccccc</item><item name="keyBorderColorBottom">#ffaaaaaa</item><item name="emoji_button_bg">?colorKeyActivated</item><item name="emoji_color">#ff000000</item></style>
+<style name="Material" parent="@style/BaseTheme"><item name="android:isLightTheme">true</item><item name="colorKeyboard">#ffeeeeef</item><item name="colorKey">@android:color/white</item><item name="colorKeyActivated">#ff2962ff</item><item name="colorLabel">@android:color/black</item><item name="colorLabelActivated">@android:color/white</item><item name="colorLabelLocked">@android:color/black</item><item name="secondaryLightOffset">0.0</item><item name="colorSubLabel">#ff555566</item><item name="keyBorderRadius">6.0dip</item><item name="keyBorderWidth">0.0dip</item><item name="keyBorderWidthActivated">0.0dip</item><item name="keyBorderColorBottom">#ff651fff</item><item name="emoji_button_bg">?colorKeyboard</item><item name="emoji_color">?colorLabel</item></style>
```

Comment:
`colorKey` on the material theme doesn't seem like a material themed keyboard.

Reply:
> `colorKey` En el tema Material no parece un teclado temático Material.

The colors are taken from a google app... In joke is to use ESL spaced at 8.2 so that it looks good... When I have time I do all that, because I've been lined up a lot (playing, and I forget)